packages/dds/merge-tree/src/test/client.rollbackFarm.spec.ts

```diff
@@ -40,7 +40,7 @@ describe("MergeTree.Client", () => {
                 mt.seedWithArray([0xDEADBEEF, 0xFEEDBED, minLength, opsPerRollback]);
 
                 // A: readonly, B: rollback, C: rollback + edit, D: edit
-                const clients = createClientsAtInitialState({ initialState: "" }, "A", "B", "C", "D");
+                const clients = createClientsAtInitialState({ initialState: "", options: {mergeTreeUseNewLengthCalculations: true} }, "A", "B", "C", "D");
```

Comment:
So anyone who uses rollback on mergeTree should set this option? If so, maybe we should set this in unit tests too.